api/src/params.ts

```diff
@@ -1,6 +1,7 @@
 export const params = {
   MASTER_ADMIN: "dappnode_admin",
-  DAPPNODE_API_URL_GET_INTERNAL_IP: "http://my.dappnode/global-envs/INTERNAL_IP",
+  DAPPMANAGER_HOSTNAMES: ["my.dappnode", "dappmanager.dappnode"],
```

Comment:
Add dappmanager container IP and comment `TODO: change dappmanager IP when migration done`